view/src/main/java/com/elpassion/android/view/InfalteHelper.kt

```diff
@@ -0,0 +1,11 @@
+package com.elpassion.android.view
+
+import android.support.annotation.LayoutRes
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+
+fun inflate(parent: ViewGroup, @LayoutRes layoutId: Int): View {
+    val inflater = LayoutInflater.from(parent.context)
+    return inflater.inflate(layoutId, parent, false)
```

Comment:
I would extract last parameter to method signature and use `false` value by default.